### DISCUSSION TITLE
Academic status verified but Copilot not active.

### DISCUSSION THREAD

**Discussion by sorenwacker** [community/community#122939] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi,

my academic status was verified a few months back, but the Copilot extension still returns an error about the description:

Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as ..."

I wonder, if I have to do anything else to use Copilot at the university? Or whether this is bug.

**Comment by davevad93** [comment]
Hi @sorenwacker ,

I don't think the issue is related to the use of Copilot at the university, as far as I know, there is no limit on the number of devices that may be used simultaneously. You could try logging out and then logging in again, but I don't think that will solve your problem because it isn't an authentication issue but a subscription one. I suggest you to contact [GitHub Support](https://support.github.com/request)

**Comment by sorenwacker** [comment]
I think it is working now. Thanks
